extensions/lite/queue/index.js

```diff
@@ -1,16 +1,34 @@
-import { EventEmitter } from 'events'
 import nanoid from 'nanoid'
+import Promise from 'bluebird'
+import _ from 'lodash'
 
 export default class Queue {
```

Comment:
No existing npm module to use in place of the custom queue?

Reply:
I searched but couldn't find a multi-dimentional one :(